inst/report/index.Rmd

```diff
@@ -389,6 +389,8 @@ StackedBarplotCovid(DataPlot100kScaleRatiosPlot %>% select(-Variable) , g_palett
 ```
 ```{r ageclasses-vacc-unvac-scale100k-tab, warning=FALSE, out.width="100%"}
 DataPlot100kScaleRatios %>%
+  mutate(value = as.character(value),
+         value = replace_na(value, "")) %>%
```

Comment:
Yes, the formatting looks still ok with small numbers. I.e. the same as before,
It is however a bit of a hack, but I could not find a way to avoid this collapsing of NA columns